mirai-core-api/src/commonMain/kotlin/contact/ContactOrBot.kt

```diff
@@ -35,9 +35,15 @@ public interface ContactOrBot : CoroutineScope {
     public val bot: Bot
 
     /**
-     * 头像下载链接
+     * 头像下载链接, 规格默认为 [AvatarSpec.LARGEST]
+     * @see getAvatarUrl
      */
     public val avatarUrl: String
-        get() = "http://q1.qlogo.cn/g?b=qq&nk=$id&s=640"
+        get() = getAvatarUrl(spec = AvatarSpec.LARGEST)
 
+    /**
+     * 头像下载链接.
+     * @param spec 头像的规格.
+     */
+    public fun getAvatarUrl(spec: AvatarSpec): String = "http://q.qlogo.cn/g?b=qq&nk=${id}&s=${spec.size}"
```

Comment:
```suggestion
    @Suppress("INAPPLICABLE_JVM_NANE")
    @JvmName("getAvatarUrl")
    public fun avatarUrl(spec: AvatarSpec): String = "http://q.qlogo.cn/g?b=qq&nk=${id}&s=${spec.size}"
```

感觉对 Kotlin 更友好（在 property 基础上加括号 spec）